cpp/CMakeLists.txt

```diff
@@ -195,18 +195,13 @@ else()
 endif(ARROW_FOUND)
 
 ###################################################################################################
-# - clone libcu++ ------------------------------------------------------------------------------------
-
-message(STATUS "Installing libcu++")
-execute_process(WORKING_DIRECTORY ${CMAKE_BINARY_DIR}
-                COMMAND git clone --branch cudf --recurse-submodules
-                        https://github.com/rapidsai/thirdparty-freestanding.git libcudacxx)
+# - copy libcu++ ------------------------------------------------------------------------------------
 
 file(MAKE_DIRECTORY "${CMAKE_BINARY_DIR}/include/libcudacxx")
 file(MAKE_DIRECTORY "${CMAKE_BINARY_DIR}/include/libcxx/include")
 
-file(INSTALL ${CMAKE_BINARY_DIR}/libcudacxx/include/        DESTINATION include/libcudacxx)
-file(INSTALL ${CMAKE_BINARY_DIR}/libcudacxx/libcxx/include/ DESTINATION include/libcxx/include)
+file(INSTALL ${LIBCUDACXX_INCLUDE_DIR} DESTINATION ${CMAKE_BINARY_DIR}/include/libcudacxx/)
+file(INSTALL ${LIBCXX_INCLUDE_DIR}     DESTINATION ${CMAKE_BINARY_DIR}/include/libcxx/include/)
```

Comment:
Why did we change the destination of these installs? This would actually install `libcudacxx` into the `include` directory previously.

Reply:
https://cmake.org/cmake/help/latest/command/file.html#install

> Relative input paths are evaluated with respect to the current source directory, and a relative destination is evaluated with respect to the current build directory. 

`CMAKE_BINARY_DIRECTORY` is defined as "The path to the top level of the build tree." Therefore, I figured that adding this as a prefix would be the same as a relative path. My aim was to make it explicit rather than implicit where the file get installed.

So both before and after this change, the files would be installed into the `include` subdirectory *of the build directory*.

Reply:
Got it, apologies I confused this block and the later `INSTALL` block which is what actually installs these files.

Reply:
No worries!